stage0/app.go

```diff
@@ -388,17 +383,17 @@ func RmApp(cfg RmConfig) error {
 			}
 		}
 
-		if err := RunCrossingEntrypoint(cfg.PodPath, cfg.PodPID, cfg.AppName.String(), appRmEntrypoint, args); err != nil {
+		if err := RunCrossingEntrypoint(cfg.PodPath, cfg.PodPID, cfg.AppName.String(), appRmEntrypoint, args, false); err != nil {
```

Comment:
we are starting to accumulate more, and more parameters. How about introducing a:
```
type CrossingEntryPoint struct {
  PodPath string
  PodPID string
  AppName string
  RmEntrypoint string
  Interactive bool
}
```
and have the following method:
```
func (cep CrossingEntryPoint) Run() error {
 ...
}
```
The caller would instantiate a struct using obvious named values, and invoke the `Run()` method.